### PR TITLE
Feature/7/privacy mds for background tasks and bibliographic

### PR DESCRIPTION
**Pull request by lzehnder**

### CODE REVIEW

File: src/BackgroundTasks/PRIVACY.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# BackgroundTasks Privacy
+Disclaimer: This documentation does not warrant completeness or correctness. Please report any missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de) or contribute a fix via [Pull Request](docs/development/contributing.md#pull-request-to-the-repositories).
+
+## General Information
+- Services and components use the BackgroundTasks service for running tasks in the background so that longer, more resource-intensive tasks do not hinder the use of ILIAS. One example is the download of large folders, which could easily prevent a user from using ILIAS for several minutes, if the BackgroundTasks service weren't being used.
+- Services and components which use the BackgroundTasks service might store, present, delete or export personal data. This is specified in their respective PRIVACY.md.
+
+## Data being stored
+- User ID which was handed over by a service or component using the BackgroundTasks service is stored.
+  - In most cases this will likely be the User ID of the account whose actions initiated the creation of the background task.
```
Comment (atoedt) [inline src/BackgroundTasks/PRIVACY.md]: Typically this will be

File: src/BackgroundTasks/PRIVACY.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- The BackgroundTasks service employs the following services, please consult the respective privacy.mds:
+  - [Filesystem](../../src/Filesystem/PRIVACY.md)
+  - [GlobalScreen](../../Services/GlobalScreen/PRIVACY.md)
+  - Setup
+  - UI
+
+## Data being presented
+- The BackgroundTasks service does not intentionaly present any personal data.
```
Comment (atoedt) [inline src/BackgroundTasks/PRIVACY.md]: The BackgroundTasks service is ignorant of the content of the title and description of a task presented to a user. The BackgroundTasks service itself does not present any personal data.

File: src/BackgroundTasks/PRIVACY.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+- If services or components which use the BackgroundTasks service write personal data into the title or description of a background task, that data would be shown to the account whose User ID has been stored for that background task.
```
Comment (atoedt) [inline src/BackgroundTasks/PRIVACY.md]: please delete

File: src/BackgroundTasks/PRIVACY.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+## Data being deleted
+- The BackgroundTasks service deletes personal data in the following cases:
```
Comment (atoedt) [inline src/BackgroundTasks/PRIVACY.md:23]: Tasks are directely deleted and do not go to the trash.

File: Modules/Bibliographic/PRIVACY.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# Bibliograpghic Privacy
+Disclaimer: This documentation does not warrant completeness or correctness. Please report any missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de) or contribute a fix via [Pull Request](docs/development/contributing.md#pull-request-to-the-repositories).
+
+## General Information
+- The Bibliographic module imports its data from a bibliography file. This data may or may not be personal in nature.
```
Comment (atoedt) [inline Modules/Bibliographic/PRIVACY.md]: like .ris or .bib

Comment (atoedt) [inline Modules/Bibliographic/PRIVACY.md]: please delete

File: Modules/Bibliographic/PRIVACY.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+- The following sections regarding the storage, presentation, deletion and export of personal information are written with the assumption that personal data has been imported. Otherwise there would be no handling of privacy-relevant data to describe at all.
```
Comment (atoedt) [inline Modules/Bibliographic/PRIVACY.md]: Can be deleted

File: Modules/Bibliographic/PRIVACY.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+## Data being stored
+- Potentially personal data from the imported bibliography file is stored as "value".
```
Comment (atoedt) [inline Modules/Bibliographic/PRIVACY.md]: please delete. 
The Bibliographic module does not store any personal data. It employs the following services, please consult the respective privacy.mds
[AccessControl](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/Services/AccessControl/PRIVACY.md)
[Info Screen Service](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/Services/InfoScreen/PRIVACY.md)

File: Modules/Bibliographic/PRIVACY.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+## Data being presented
+- Persons with "Read" permission on the Bibliography object are presented with data which was stored as "value".
```
Comment (atoedt) [inline Modules/Bibliographic/PRIVACY.md]: delete

File: Modules/Bibliographic/PRIVACY.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+- This data is shown in the entries of the "Content" tab as well as in the "Detail View" of an entry, which can be opened by clicking on the entry.
```
Comment (atoedt) [inline Modules/Bibliographic/PRIVACY.md:13]: The User name of the person who created the Biliographic  is shown in Permission Tab > owner  and on the Info Tab.

Comment (atoedt) [inline Modules/Bibliographic/PRIVACY.md]: delete 
Does not apply.

File: Modules/Bibliographic/PRIVACY.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+## Data being deleted
+- Persons with "Delete" permission on the Bibliography object can delete the Bibliography object and therefore data which was stored as "value".
```
Comment (atoedt) [inline Modules/Bibliographic/PRIVACY.md]: delete 
Does not apply.

File: Modules/Bibliographic/PRIVACY.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+  - If the Trash is activated, deletion from the Repository merely pushes the Bibliography object to the Trash. 
+  - In the latter case the Bibliography object and its data will only be deleted at last when:
+    - Manually emptying the Trash at "Administration" > "System Settings and Maintenance" > "Repository Trash and Permissions" > "Trash". 
+    - Or running a Cronjob for emtpying the Trash.
+- Persons with "Edit Settings" permission on the Bibliography object can cause the deletion of data which was stored as "value" by using the "Override Entries" function.
+  - This can be done by activating the "Override Entries" option in the "Settings" tab, selecting a bibliography file and clicking "Save".
+  - The resulting update of the Bibliography object with data from the new bibliography file can lead to data being overwritten, which amounts to the same thing as this data being deleted.
+
+## Data being exported
+- Persons with "Edit Settings" permission on the Bibliography object can export data which was stored as "value" by using the export function in the "Export" tab.
```
Comment (atoedt) [inline Modules/Bibliographic/PRIVACY.md]: delete 
Does not apply.

**Comment by chfsx** [comment]
Thanks a lot @lzehnder and @atoedt !

**Pull request merged**